var/spack/repos/builtin/packages/lynx/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Lynx(AutotoolsPackage):
+    """Lynx is the text web browser."""
+
+    homepage = "https://lynx.invisible-island.net/"
+    url      = "https://invisible-mirror.net/archives/lynx/tarballs/lynx2.8.9rel.1.tar.gz"
+
+    version('2.8.9rel.1', sha256='a46e4167b8f02c066d2fe2eafcc5603367be0e3fe2e59e9fc4eb016f306afc8e')
+    version('2.8.8rel.1', sha256='28cdb16136eea8ca097d2994138519bfbbdfe63b7d10fee6a4908580e5f55fc5')
+    version('2.8.7rel.1', sha256='6570e3088c0ae22fbd48a528f4841a1f2b83f588c7d31c059c3bbbcd5c7e7298')
```

Comment:
1. Should we use this autodetected versions or something like `2.8.9.1` and code `url_for_version`?
2. Can you double check version 2.8.8 and 2.8.7? On the website the latest versions seem to be `2.8.8rel2` and `2.8.7rel2`.

Reply:
1. done in https://github.com/spack/spack/pull/17099/commits/ef18e8476e6c6c9e42667334a5d5caee7f117c8b
2. `2.8.8.2` compile failed, so I only keep `2.8.9.1` in my new patch